web/page0/src/test_apis.py

```diff
@@ -0,0 +1,330 @@
+def info():
+    sleep(0.1)
+
+    return mem["info"]
+
+
+# GET /v1/zones
+@apis.route("/api/v1/zones", methods=["GET"])
+def zones():
+    sleep(0.1)
+    zones = list(mem["zones"].keys())
+    return {"items": zones}
+
+
+# GET /v1/zones/:zone/hosts
+@apis.route("/api/v1/zones/<zone>/hosts", methods=["GET"])
+def get_hosts(zone):
+    sleep(0.1)
+    hosts = []
+    for host in mem["zones"][zone]:
+        hosts.append({"name": host["name"], "gcp": host["gcp"]})
+    return {"items": hosts}
+
+
+# POST /v1/zones/:zone/hosts
+@apis.route("/api/v1/zones/<zone>/hosts", methods=["POST"])
+def post_host(zone):
+    sleep(0.1)
+    body = flask.request.json
+
+    def task():
+        mem["zones"][zone].append(
+            {
+                "name": gen_host_name(5),
+                "gcp": body["host_instance"]["gcp"],
+                "groups": [],
+            }
+        )
+
+    bg_run(task, 1)
+
+    return {"name": gen_operation_name(15), "done": False}
+
+
+# DELETE /v1/zones/:zone/hosts/:host
+@apis.route("/api/v1/zones/<zone>/hosts/<hostname>", methods=["DELETE"])
+def delete_host(zone, hostname):
+    sleep(0.1)
+    _, idx = find_host(mem["zones"][zone], hostname)
+
+    if idx < 0:
+        flask.abort(404)
+
+    def task():
+        mem["zones"][zone].pop(idx)
+    
+    bg_run(task, 1)
+
+    return {"name": gen_operation_name(15), "done": False}
+
+
+# GET /v1/zones/:zone/hosts/:host/groups
+@apis.route("/api/v1/zones/<zone>/hosts/<hostname>/groups", methods=["GET"])
+def get_groups(zone, hostname):
+    sleep(0.1)
+    host, _ = find_host(mem["zones"][zone], hostname)
+    if not host:
+        flask.abort(404)
+
+    return {"groups": host["groups"]}
+
+
+# DELETE /v1/zones/:zone/hosts/:host/groups
+@apis.route(
+    "/api/v1/zones/<zone>/hosts/<hostname>/groups/<groupname>", methods=["DELETE"]
+)
+def delete_group(zone, hostname, groupname):
+    sleep(0.1)
+    host, hostidx = find_host(mem["zones"][zone], hostname)
+
+    if not host:
+        flask.abort(404)
+
+    _, groupidx = find_group(host["groups"], groupname)
+
+    if groupidx < 0:
+        flask.abort(404)
+
+    def task():
+        mem["zones"][zone][hostidx]["groups"].pop(groupidx)
+
+    bg_run(task, 10)
+
+    return {"name": gen_operation_name(15), "done": False}
+
+
+# POST /v1/zones/:zone/hosts/:host/cvds
+@apis.route("/api/v1/zones/<zone>/hosts/<hostname>/cvds", methods=["POST"])
+async def post_group(zone, hostname):
+    sleep(1)
+    host, hostidx = find_host(mem["zones"][zone], hostname)
+
+    if not host:
+        flask.abort(404)
+
+    body = flask.request.json
+    group = body["group"]
+
+    def task():
+        mem["zones"][zone][hostidx]["groups"].append(group)
+        print(mem["zones"][zone][hostidx]["groups"])
+    
+    bg_run(task, 3)
+    
+    return {"name": gen_operation_name(15), "done": False}
+
+
+"""
+Mock server-only APIs 
+"""
+
+
+@apis.route("/api/reset", methods=["GET"])
+def reset():
+    init()
+
+
+"""
+Main
+"""
+init()
```

Comment:
If here is Main(), can you please write a code like below?

if __name__ == "__main__":
&nbsp;&nbsp;&nbsp;&nbsp;init()